snap/info.go

```diff
@@ -275,6 +275,27 @@ func (s *Info) NeedsClassic() bool {
 	return s.Confinement == ClassicConfinement
 }
 
+// RenamePlug renames the plug from oldName to newName, if present
+func (s *Info) RenamePlug(oldName, newName string) {
```

Comment:
This should be `renamePlug` inside `broken.go` rather than a public API. This is far from being a well supported or desired operation.